app/change_sets/hyrax/collection_change_set.rb

```diff
@@ -36,26 +36,36 @@ def select_files
       Hash[all_files_with_access]
     end
 
-    # Terms that appear above the accordion
-    def primary_terms
-      [:title]
-    end
+    class << self
+      # Combined primary and secondary terms
+      def terms
```

Comment:
what is this for? Why do we need it?

Reply:
Just trying to green up a failing test, the spec was failing on `described_class.terms` with a `NoMethodError` but we can get of it and re-work the test if needed.

Reply:
I believe that would be taken care of if you ported `autocreate_fields!` into CollectionsChangeSet, but I'm really not sure if `fields` has any utility anymore.

Reply:
Ok cool, should we just merge this PR now to keep it moving and talk about `fields` with the group after standup tomorrow?